crates/rolldown/src/ast_scanner/mod.rs

```diff
@@ -717,6 +719,24 @@ impl<'me, 'ast: 'me> AstScanner<'me, 'ast> {
     }
   }
 
+  pub fn try_extract_parent_static_member_expr_chain(
+    &self,
+    max_len: usize,
+  ) -> Option<(Span, Vec<CompactStr>)> {
+    let mut span = SPAN;
+    let mut props = vec![];
+    for ancestor_ast in self.visit_path.iter().rev().take(max_len) {
+      match ancestor_ast {
+        AstKind::MemberExpression(MemberExpression::StaticMemberExpression(expr)) => {
+          span = ancestor_ast.span();
```

Comment:
This relies on `EnsureUniqueSpan` to ensure uniqueness span of all nodes. Don't forget to change it too.

Reply:
Not sure if we need that, since we just ignore `unspanned` before https://github.com/rolldown/rolldown/pull/2703/files#diff-e892c2c30c0c3ce0005c3ca28e5f0ad7692f8dcc64dcdb79b32898abcfebdfdbL86, and I did the same thing now.